index.js

```diff
@@ -9,9 +9,30 @@ const modules = {
                 'type': 'callback',
                 'name': 'list_accounts',
                 'args': [
-                    { 'name': 'page', 'type': 'number' },
-                    { 'name': 'item', 'type': 'number' },
-                    { 'name': 'order', 'type': 'number' },
+                    {
+                        'name': 'page',
+                        'type': 'number',
+                        'attr': {
+                            'proto': '',
+                            'message': '',
+                        }
```

Comment:
I think you can delete the field directly without the proto type, don’t need to leave it blank